server/build/index.js

```diff
@@ -1,11 +1,14 @@
 import { tmpdir } from 'os'
 import { join } from 'path'
 import fs from 'mz/fs'
+import fs2 from 'fs'
```

Comment:
Maybe, `import { createWriteStream, createWriteStream } from 'fs'`, and move up along with the other native modules.

Reply:
Sounds reasonable!